src/hooks/sceneGraphNodes/sceneGraphNodes.ts

```diff
@@ -0,0 +1,86 @@
+import { useMemo } from 'react';
+
+import { useAppSelector } from '@/redux/hooks';
+import { PropertyOwner, SceneGraphNodeGuiSettings, Uri } from '@/types/types';
+
+import { SceneGraphNodesFilters } from './types';
+import {
+  isSgnFocusable,
+  isSgnHiddenInGui,
+  sgnGuiOrderingNumber,
+  sgnGuiPath
+} from './util';
+
+export function useIsSgnFocusable(uri: Uri): boolean {
+  return (
+    useAppSelector((state) => isSgnFocusable(uri, state.properties.properties)) || false
+  );
+}
+
+export function useSceneGraphNodes({
+  showHiddenNodes = false,
```

Comment:
Is it possible here to use the default settings that are stored in `src/panels/Scene/SceneTree`? Is that something we want?

Reply:
No, I don't think so. The deafult setting for the filtering could be different for different use cases. 

For example, in the scene menu we show non-focusable nodes per default, but in the navigation menu we don't